content/src/patterns/organizing-work/backlog.md

```diff
@@ -21,11 +25,10 @@ Types of backlog include:
 
 Each item on a (prioritized) backlog typically contains:
 
--   a **short description** of a [deliverable](glossary:deliverable) or a [driver](glossary:organizational-driver)
--   a **unique reference number** (or link) for each work item
--   (the **order of work items**)
--   **dependencies** to other work items or projects
--   **due date** (if necessary)
--   (optional) a measure for **value**
--   (optional) a measure for **investment** (often an estimate of time or complexity)
+-   a **short description** of the work item typically a [deliverable](glossary:deliverable) [requirement](glossary:requirement) or a [driver](glossary:organizational-driver)
+-   reference to other **interdependent work items** or projects, as well as to any other relevant information
```

Comment:
add emphasis in "other relevant information", because that may be as important as interdependent work items